modules/credential/src/main/java/org/eclipse/xpanse/modules/credential/CredentialCenter.java

```diff
@@ -221,7 +221,8 @@ public AbstractCredentialInfo getCredential(Csp csp,
                 csp, credentialType, userId);
         if (credentialInfos.isEmpty()) {
             throw new CredentialsNotFoundException(
-                    String.format("No credential information found for the given Csp:%s.", csp));
+                    String.format("No credential information found for the given Csp:%s.",
+                            csp.toValue()));
```

Comment:
There are other uses that need to be modified in the class `PluginManager`.

Reply:
@baixinsui thanks for the hint. I have now corrected PluginManager class as well